tests/e2e/test_api.py

```diff
@@ -16,26 +16,25 @@
     INVALID_QUERY_RESP,
 )
 from ols.utils import suid
-from tests.e2e import (
+from tests.e2e.utils import (
     cluster_utils,
     helper_utils,
     metrics_utils,
 )
-from tests.e2e.constants import (
+from tests.e2e.utils.constants import (
     BASIC_ENDPOINTS_TIMEOUT,
     CONVERSATION_ID,
     LLM_REST_API_TIMEOUT,
     NON_LLM_REST_API_TIMEOUT,
 )
-from tests.scripts.must_gather import must_gather
-from tests.scripts.validate_response import ResponseEvaluation
-
-from .postgres_utils import (
+from tests.e2e.utils.decorators_utils import retry
```

Comment:
we don't have to repeat "utils" two times probably. For example in Go ecosystem it is not allowed.

Reply:
nice!